core/domain/state_domain.py

```diff
@@ -916,8 +938,946 @@ def get_all_outcomes(self) -> List[Outcome]:
             outcomes.append(self.default_outcome)
         return outcomes
 
+    def _validate_continue_input(self) -> None:
+        """Validates the Continue interaction.
+
+        Raises:
+            ValidationError. Raises a validation error in the following
+                condition:
+                - Text value is empty or is more than 20 characters.
+                - Answer group is present.
```

Comment:
No, add a separate line for each validation error, like this:
```suggestion
            ValidationError. Text value is empty or is more than 20 characters.
            ValidationError. Answer group is present.
```

ditto elsewhere

Reply:
Done